packages/world/src/ObjectTypes.sol

```diff
@@ -224,6 +224,7 @@ library ObjectTypes {
   // Misc
   // ------------------------------------------------------------
   ObjectTypeId constant Player = ObjectTypeId.wrap(Misc | 0);
+  ObjectTypeId constant ForceFieldFragment = ObjectTypeId.wrap(Misc | 1);
```

Comment:
Added this to the `objects.ts` file, from which this file is generated: https://github.com/tenetxyz/biomes-client/commit/afd77bc6da9af9a89f2a154421d98fb6964641ac